Gemfile

```diff
@@ -40,11 +40,11 @@ gem 'jbuilder'
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 # Reduces boot times through caching; required in config/boot.rb
-gem 'bootsnap', require: false
 
-gem 'cancancan'
-gem 'devise'
 gem 'factory_bot_rails'
+
+gem 'bootsnap', require: false
+
 # Use Sass to process CSS
 # gem "sassc-rails"
 
```

Comment:
- [ ] As mentioned by the previous reviewer kindly add `gem 'devise'` in your Gemfile as it's causing migration error while trying to set up the app.